lib/generators/solidus_frontend/install/install_generator.rb

```diff
@@ -13,6 +13,19 @@ def copy_initializer
         template 'initializer.rb', 'config/initializers/solidus_frontend.rb'
       end
 
+      def robots_directives
+        append_file "public/robots.txt", <<-ROBOTS.strip_heredoc
+          User-agent: *
+          Disallow: /checkout
+          Disallow: /cart
+          Disallow: /orders
+          Disallow: /user
+          Disallow: /account
+          Disallow: /api
+          Disallow: /password
```

Comment:
Do we have this endpoint?

Reply:
we do, although it's provided by solidus_auth_devise